frontend/dispatcher.lua

```diff
@@ -112,6 +112,7 @@ local settingsList = {
     zoom = { category="string", event="SetZoomMode", title=_("Zoom to"), args={"contentwidth", "contentheight", "pagewidth", "pageheight", "column", "content", "page"}, toggle={"content width", "content height", "page width", "page height", "column", "content", "page"}, paging=true,},
 
     -- parsed from CreOptions
+    -- the rest of the table elements are built from their counterparts in CreOptions
```

Comment:
But where ? I didn't find it when I was looking for it :)

Reply:
https://github.com/koreader/koreader/blob/4e5def4282ebafe4eb0d089ecdd6a4af5448f96e/frontend/dispatcher.lua#L51-L93